src/tests/src/test_system.cpp

```diff
@@ -15,10 +16,10 @@ TEST(TestSystems, TestChatSystemSendMsg) {
     SystemManager_Mock mock(entitySystem);
     ChatSystem chat(mock);
     EntityManager man;
-    CMapClient_Mock cli;
+    CMapClient_Mock cli(new Core::CNetwork_Asio);
```

Comment:
This is needed as when you clean up this test, the test will segfault due to nullptrs